ui/docs-app/docs_app/theme/assets/semantic-ui/js/docs_app/forms/components/IdentifiersField/IdentifiersField.jsx

```diff
@@ -4,6 +4,22 @@ import { ArrayField, SelectField, TextField } from "react-invenio-forms";
 import { i18next } from "@translations/docs_app/i18next";
 import { ArrayFieldItem } from "@js/oarepo_ui";
 
+export const objectIdentifiersSchema = [
+  { value: "DOI", text: "DOI" },
+  { value: "Handle", text: "Handle" },
+  { value: "ISBN", text: "ISBN" },
+  { value: "ISSN", text: "ISSN" },
+  { value: "RIV", text: "RIV" },
+];
+
+export const systemIdentifiersSchema = [
```

Comment:
Is `objectIdentifiersSchema` and `systemIdentifiersSchema` used anywhere inside component?

Reply:
@mirekys The schemas are used as options list for the system/object identifier component (IdentifiersField). The schemas (options lists) used to be in DepositForm component, but as we are using identifiers field in multiple places, I thought it made sense to colocate the schemas with the Field itself, so if I import the field somewhere, I can also import the needed schemas from the same file. It did not make sense to be importing schemas from the form component (DepositForm). However, if we don't like it this way, I can return it to deposit form. Please let me know.

Reply:
@mirekys Actually I see that we also have another options list in CreatibutorsField https://github.com/Narodni-repozitar/nr-docs/blob/bd5ba98a320f2e3b7d1ec724ae1822529578b110/ui/docs-app/docs_app/theme/assets/semantic-ui/js/docs_app/forms/components/CreatibutorsField/CreatibutorsIdentifiers.jsx#L27

I am not sure if the listing from this component 

const authorityIdentifiersSchema = [
  { value: "orcid", text: "ORCID" },
  { value: "scopusID", text: "ScopusID" },
  { value: "researcherID", text: "ResearcherID" },
  { value: "czenasAutID", text: "CzenasAutID" },
  { value: "vedidk", text: "vedIDK" },
  { value: "institutionalID", text: "InstitutionalID" },
  { value: "ISNI", text: "ISNI" },
  { value: "ROR", text: "ROR" },
  { value: "ICO", text: "ICO" },
  { value: "DOI", text: "DOI" },
];

is the full exhaustive list of "system identifiers". If yes, with permission I would move it to be with identifiers field (to replace the systemIdentifiersSchema there and in that way we are not duplicating this options list in multiple places. Let me know what you think.